examples/rainier/main.bal

```diff
@@ -352,11 +353,11 @@ public function main() returns error? {
     rainier:Client rainierClient = check new ();
 
     check checkBuilding(rainierClient);
+    runtime:sleep(60);
```

Comment:
Shall we split the small sleep period into each check

Reply:
Reduced this time to 15 seconds. Is it okay now?

Reply:
is it fail without sleep?

Reply:
Yes

Reply:
Since we run two examples using google sheets one after each other, it might be better to give a sleep here. But probably we might be able to reduce it to 30 seconds

Reply:
Is it possible the add small sleep time(5s) after each test function